From 2ee5f18340e362965e6b99a12a41e5f2ac7d21db Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Mon, 7 Nov 2011 16:46:56 +0530
Subject: [PATCH 0389/1092] AM335XEVM: Update mach-types

New machine-id added, obtained from RMK.

Note: Please note that, the allocated mach-type has typo-mistake
(am335evm => am335xevm) which needs to be corrected. This is going
to be custom patch, since this will get enrolled by Russell automatically.

Signed-off-by: Afzal Mohammed <afzal@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/tools/mach-types | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/tools/mach-types b/arch/arm/tools/mach-types
index ccbe16f..ead93da 100644
--- a/arch/arm/tools/mach-types
+++ b/arch/arm/tools/mach-types
@@ -1126,3 +1126,4 @@ atdgp318		MACH_ATDGP318		ATDGP318		3494
 m28evk			MACH_M28EVK		M28EVK			3613
 smdk4212		MACH_SMDK4212		SMDK4212		3638
 smdk4412		MACH_SMDK4412		SMDK4412		3765
+am335xevm		MACH_AM335XEVM		AM335XEVM		3589
-- 
1.7.11.2

